scripts/generator/graphql_generator/csharp/SDK/iOS/ApplePayEventReceiverBridge.cs.erb

```diff
@@ -0,0 +1,32 @@
+#if UNITY_IPHONE
+namespace <%= namespace %>.SDK.iOS {
+    using System.Collections;
+    using System.Collections.Generic;
+    using UnityEngine;
+
+    public class ApplePayEventReceiverBridge : MonoBehaviour, IApplePayEventReceiver {
+
+        public IApplePayEventReceiver Receiver;
+
+        public ApplePayEventReceiverBridge(IApplePayEventReceiver receiver) {
```

Comment:
Since this is instantiated via `AddComponent` this constructor is never called. We should 🔥